app/controllers/gems_adoptions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class GemsAdoptionsController < ApplicationController
+  
+  def index
+    @gems_adoptions = GemsAdoption.all
+  end
+
+  def create
+    @gems_adoption = GemsAdoption.new
+    @gems_adoption.gem = params[:name]
+    @gems_adoption.description = params[:info]
+    @gems_adoption.user_id = current_user.id
+    @gems_adoption.save
```

Comment:
This will probably require you to use StrongParameters to whitelist the parameters: http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters

Let us know if we can help you with that.